### ISSUE TITLE
Feature Request: biblio_type should be an entity

### ISSUE THREAD

**Issue by wundo** [Gizra/biblio#89]
IMO biblio_type should be an exportable entity, so we can use Entity API to leverage exporting biblio_types to code.

What do you think?

**Comment by amitaibu** [comment]
Maybe, although we could have features integration what needing to expose it to the entity system

**Comment by wundo** [comment]
Would Entity API integration be a lower level integration and just a few lines of code?

**Comment by amitaibu** [comment]
Yeah, I guess it would be quicker to leverage entity API. PRs are welcome :)

btw, are you already using this Biblio branch?
